core/integer/round_spec.rb

```diff
@@ -74,4 +74,18 @@
     obj.stub!(:to_int).and_return([])
     lambda { 42.round(obj) }.should raise_error(TypeError)
   end
+
+  ruby_version_is "2.4" do
+    it "returns different rounded values depending on the half option" do
+      25.round(-1, half: :up).should      eql(30)
+      25.round(-1, half: :down).should    eql(20)
+      25.round(-1, half: :even).should    eql(20)
+      35.round(-1, half: :up).should      eql(40)
+      35.round(-1, half: :down).should    eql(30)
+      35.round(-1, half: :even).should    eql(40)
+      (-25).round(-1, half: :up).should   eql(-30)
+      (-25).round(-1, half: :down).should eql(-20)
+      (-25).round(-1, half: :even).should eql(-20)
+    end
```

Comment:
It would be interesting to also test for rounding values > 0, could you add a few examples like `12.round(1)`?